.travis.yml

```diff
@@ -11,12 +11,45 @@ cache:
     - $HOME/.gradle/wrapper/
 android:
   components:
-    - platform-tools
     - tools
-    - build-tools-25.0.0
+    - build-tools-25.0.1
     - android-25
-
-    # Extras
-    - extra-google-m2repository
+    - platform-tools
     - extra-android-m2repository
-script: ./gradlew clean :library:testAll :library:prepareArtifacts
```

Comment:
That's the command I run before uploading to jCenter, I want to run it on each build to make sure that the build will be "releasable".